source/2019-07-19-the-ember-times-issue-107.md

```diff
@@ -79,12 +79,13 @@ We encourage you to [read them today](https://dev.to/mitchartemis/making-an-elec
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [Improving Your DX on Ember 💯](https://dockyard.com/blog/2019/06/10/improving-your-dx-on-ember-part-i-quick-wins)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@Serabe](https://github.com/Serabe) has a series on the Dockyard blog about how to improve your **Developer Experience (DX)**. Check it out for happier Ember-ing! 
```

Comment:
```suggestion
[@Serabe](https://github.com/Serabe) has a series on the DockYard blog about how to improve your **Developer Experience (DX)**. Check it out for happier Ember-ing! 
```
I think DockYard is title-cased. Or whatever case that is. Otherwise looks ✨

Reply:
thanks!